src/Files.App/UserControls/Widgets/FileTagsWidget.xaml

```diff
@@ -89,6 +89,7 @@
 					</Grid>
 
 					<!--  Contents  -->
+					<!--  We hide the scrollbar because it may cause a crash inside the ScrollViewer (#12370)  -->
```

Comment:
```suggestion
					<!--  The Vertical Scrollbar Must Be Hidden Because It May Cause a Crash Inside The ScrollViewer (#12370)  
  -->
```

Reply:
@0x5bfa why do you put a line break before `-->`?

Reply:
That's my fault. Don't put it.